quickwit/quickwit-doc-mapper/src/default_doc_mapper/default_mapper.rs

```diff
@@ -535,6 +544,18 @@ impl DocMapper for DefaultDocMapper {
 
         if let Some(dynamic_field) = self.dynamic_field {
             if !dynamic_json_obj.is_empty() {
+                if !self.concate_dynamic_fields.is_empty() {
+                    let json_obj_values = dynamic_json_obj
+                        .iter()
+                        .flat_map(|(_key, val)| JsonValueIterator::new(val.clone()))
```

Comment:
```suggestion
                        .values()
                        .flat_map(|child_json_value| JsonValueIterator::new(child_json_value.clone()))
```

Reply:
Would it have been easier to build a `serde_json::Value` and pass it to `JsonValueIterator::new`?